### DISCUSSION TITLE
添加腾讯地图的组件VcImageryProviderTencent有没有可提供选择显示和隐藏POI点的办法？

### DISCUSSION THREAD

**Discussion by HHKMFZY** [zouyaoji/vue-cesium#617] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/e546f421-5371-4473-b8e2-eda2bc4d9e88)
如图：
![image](https://github.com/user-attachments/assets/44b27a37-5e0b-4bb3-b766-570cb83ae1d7)
我想把这些地点名称的标注隐藏，但该组件似乎没有提供可选的参数。类似于腾讯地图官方API的示例：
![image](https://github.com/user-attachments/assets/5788815e-3839-4e17-9349-695f814ed71a)

**Comment by zouyaoji** [comment, marked as answer]
不支持，官方估计是矢量瓦片。这儿接的是图片的，有本质区别。
